ember-cli-build.js

```diff
@@ -15,31 +15,6 @@ module.exports = function (defaults) {
       cascade: true,
       sourcemap: true,
     },
-    autoImport: {
-      webpack: {
-        node: {
-          global: true,
-          __filename: true,
-          __dirname: true,
-        },
-        module: {
-          rules: [{ test: /\.handlebars$/, loader: 'handlebars-loader' }],
```

Comment:
Not sure if this is still used somewhere?

Reply:
I honestly have no idea what this does/did